app/controllers/notification_subscriptions_controller.rb

```diff
@@ -18,9 +18,7 @@ def upsert
       @notification_subscription.notifiable.update(receive_notifications: false) if current_user_is_author?
     else
       @notification_subscription.config = params[:config] || "all_comments"
-      if params[:config] == "all_comments" && current_user_is_author?
-        @notification_subscription.notifiable.update(receive_notifications: true)
-      end
+      @notification_subscription.notifiable.update(receive_notifications: true) if params[:config] == "all_comments" && current_user_is_author?
```

Comment:
I'm not a mega fan of the giant ifs at the end, i think the code was more readable before. I'm thinking we should not enforce this, what do you think?

Reply:
Agree, I haven't checked the autocorrected stuff properly.